modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -524,6 +525,10 @@ private ObjectField _addObjectField(
 			}
 		}
 
+		_objectFieldSettingLocalService.addObjectFieldSetting(
+			user.getUserId(), objectField.getObjectFieldId(),
+			"objectDefinition1ShortName", objectDefinition1.getShortName());
```

Comment:
Could you please replace the string for the constant that you created in ObjectFieldSettingConstants.java?